gsms/gsms.py

```diff
@@ -474,10 +471,11 @@ def main() -> None:
                 if has_app(sunshine_apps=sunshine_apps, name=name):
                     continue
 
-                # Increase count here to exclude some stuff
-                count += 1
-
                 cmd = application.find("StreamingCommandLine").text
+                if cmd is None:
+                    print(application.find("DisplayName").text, 'has no streaming command line. Skipping')
+                    continue
```

Comment:
You don't want to notify users about which are getting skipped? Personally I like to know that kind of thing so I'm not wondering why something didn't get added.

Reply:
It's fine to have a message, personally I think it would clutter up the stdout.

Ideally, logging should probably be implemented with different levels, and this would go under debug logging... but that would require a separate PR.

Anyway, I updated the suggestion.

Reply:
I am with ReengineArcher here. Ideally id love to know how many games can even have no cmd. We also didn't inform people about non streamable games being skipped before. Ultimately the application adds everything that can be added which would include it has some executable cmd. Maybe in the future there can be a log level PR so people can choose what to print to stdout

Reply:
OK, it's fine with me if you want to remove the user notification (again) then.